data/schemas/files.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Files",
+    "description": "Language specific list of source textual files",
+    "type": "array",
+    "items": {
+        "additionalProperties": false,
+        "required": [
+            "category",
+            "textname",
+            "filenr"
+        ],
+        "properties": {
+            "category": { "type": "string" },
+            "textname": { "type": "string" },
+            "filename": { "type": "string" },
+            "filenr": { "type": "integer" },
+
+            "link": { "type": "string" },
+            "link2": { "type": "string" },
+            "source": { "type": "string" },
+            "alt_filename": { "type": "string" },
+            "displayName": { "type": "string" },
+
+            "chapters": { 
+                "type": "array",
+                "description": "Only for Sanskrit???"
```

Comment:
we never followed through with the chapter idea (it would be too much to annotate...)